zds/tutorialv2/views/display.py

```diff
@@ -1,5 +1,6 @@
 import logging
 
+from django.db.models import F
 from django.conf import settings
 from django.http import Http404
 from django.utils.translation import gettext_lazy as _
```

Comment:
Mon perfectionnisme préfère l'ordre alphabétique :P 
```suggestion
from django.conf import settings
from django.db.models import F
from django.http import Http404
from django.utils.translation import gettext_lazy as _
```

Reply:
oh haha, comment modifier ça du coup ?

Reply:
Normalement tu as un bouton pour accepter la suggestion de mon commentaire